### PR TITLE
Empty blocks are now rendered as <br> instead of <p><br></p>

### PR DESCRIPTION
**Pull request by Vall3y**

### CODE REVIEW

File: src/stateToHTML.js [hunk 1]
```diff
@@ -217,7 +217,10 @@ class MarkupGenerator {
       this.currentBlock += 1;
       return;
     }
-    this.writeStartTag(blockType, customRenderer, block.data);
+
+    if (block.text.trim() !== '' || isEmptyBlock(blockType)) {
```
Comment (Vall3y) [inline src/stateToHTML.js]: If the text of the block is not empty, or it is an empty block type (such as `<hr>`), then the start tag should be rendered.

File: src/stateToHTML.js [hunk 2]
```diff
@@ -217,7 +217,10 @@ class MarkupGenerator {
...
+      this.writeStartTag(blockType, customRenderer, block.data);
+    }
 
     if (!isEmptyBlock(blockType)) {
       this.output.push(this.renderBlockContent(block));
```
Comment (Vall3y) [inline src/stateToHTML.js:229]: The block content is rendered either way (in the renderBlockContent, the <br> is rendered. So it needs to be rendered either way)

**Comment by Vall3y** [comment]
This adds complexity to the code, but at its current state (the repo was a bit messy as we picked it up), I can't think of a way more simple than that to solve this problem. Let me know what you think.

File: src/stateToHTML.js (hunk 1 above)
Comment (elektronik2k5) [inline src/stateToHTML.js]:
``` js
const isEmptyText = block.text.trim().length === 0;
const isCurrentBlockEmpty = isEmptyBlock(blockType);
```

then use these twice: here and in `this.writeEndTag(blockType);`

**Review by elektronik2k5 — CHANGES_REQUESTED** [review]

**Comment by Vall3y** [comment]
@elektronik2k5 done

**Review by elektronik2k5 — APPROVED** [review]

**Pull request merged**